lib/auth/auth.go

```diff
@@ -1169,6 +1169,12 @@ func (a *Server) Close() error {
 		errs = append(errs, err)
 	}
 
+	if a.Services.AuditLogSessionStreamer != nil {
+		if err := a.Services.AuditLogSessionStreamer.Close(); err != nil {
+			errs = append(errs, err)
+		}
+	}
+
```

Comment:
this change is needed, because before Close on AuditLog was never called, now we call it explicitly.

Reply:
"Fixes #15210" then?

Reply:
yes, exactly